pkg/scm/github/context.go

```diff
@@ -113,3 +113,7 @@ func (c *Context) HasExecutedActionGroup(name string) bool {
 func (c *Context) AllowPipelineFailure(ctx context.Context) bool {
 	return len(c.PullRequest.findModifiedFiles(state.ConfigFilePath(ctx))) == 1
 }
+
+func (c *Context) GetCodeOwners() []scm.Actor {
+	return []scm.Actor{}
```

Comment:
maybe should return err or log its unsupported?

Reply:
this is fine; GitHub doesn't work at all, so will need a bunch of work eventually to get it working :)